### ISSUE TITLE
wasm-ld: error: unknown argument: --ltoO3

### ISSUE THREAD

**Issue by UMU618** [EOSIO/eosio.cdt#222]
```
eosio-cpp -contract=${contract} -abigen ${contract}.cpp -o ${contract}.wasm -O=3 -lto-opt=O3
/usr/local/eosio.cdt/bin/wasm-ld: error: unknown argument: --ltoO3
```

`eosio-cpp -help` shows:

`-lto-opt=<string>        - LTO Optimization level (O0-O3)`

Succeeded: 

`eosio-cpp -contract=${contract} -abigen ${contract}.cpp -o ${contract}.wasm -O=3 -lto-opt=-O3`

**Comment by larryk85** [comment]
Side note for the `-O=3` you can simply use `-O3` like a normal compiler.

**Comment by larryk85** [comment]
Resolved by PR #233
